codice/ddf
# DDF-04475: Refactor result-filter and result-filter.list to React

#### What does this PR do?
Refactors existing `result-filter` and `result-filter.list` to React
#### Who is reviewing it? 
@bdeining 
@jrnorth 
@andrewkfiedler 
@djblue 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining

#### How should this be tested?
1. Perform search
2. Add filter using filter widget (should look and function as before)

1. Create list
2. Perform search
3. Add list filter (should look and function as before)

#### Any background context you want to provide?
#### What are the relevant tickets?
For GH Issues:
Fixes: #4475 

#### Screenshots
Query:
<img width="1792" alt="screen shot 2019-03-01 at 6 12 36 pm" src="https://user-images.githubusercontent.com/118054/53672022-da5aa500-3c4e-11e9-9f73-ad931bd70b6e.png">

List:
<img width="1792" alt="screen shot 2019-03-01 at 6 13 01 pm" src="https://user-images.githubusercontent.com/118054/53672023-da5aa500-3c4e-11e9-922c-03ae7e2275d0.png">

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
